### DISCUSSION TITLE
Add plots to a single PDF

### DISCUSSION THREAD

**Discussion by kinnetekLluis** [oxyplot/oxyplot#2011] [Q&A] [answered]
Sorry for being annoying and asking 2 questions in a row.

I have some oxyplot plots generated and I would like to add them to a single pdf. 

My idea is to have them in Landscape orentation and one plot per page (full size if possible). Can be done directly or should I look for another tool?

I let you 2 random images so you dont need to look for them.

Thanks in advance!

![PieDemo](https://github.com/oxyplot/oxyplot/assets/113010916/6ad05b4e-1d60-445b-90c0-702c18841c5a)
![ScatterDemo](https://github.com/oxyplot/oxyplot/assets/113010916/be2f65f3-3a26-4c05-92b3-73aa6fc9fcca)

**Comment by VisualMelon** [comment, marked as answer]
This isn't supported by any of the exporters at current, but should be easy enough to bodge. Within each exporter, there is a point where they run `model.Render(rc, new OxyRect(0, 0, width, height));`, and if you're happy to copy chunks of the implementation you can change this to plot whatever you want, e.g. two models, and can change the top-left coordinate to position them however you want.

Linking #55 and #391

**Reply by VisualMelon** [reply]
In the past, I have used PdfSharp to assemble plots into larger figures, but I'm not sure what the state of that project is.
